Emby.Server.Implementations/Dto/DtoService.cs

```diff
@@ -621,6 +621,7 @@ private void AttachPeople(BaseItemDto dto, BaseItem item)
                 {
                     baseItemPerson.PrimaryImageTag = GetTagAndFillBlurhash(dto, entity, ImageType.Primary);
```

Comment:
huh, why not `GetTagAndFillBlurhash(baseItemPerson, ...)` then?

Reply:
`BaseItemPerson` doesn't inherit from `BaseItemDto` so it can't be passed into that function